Cargo.toml

```diff
@@ -19,6 +19,7 @@ indicatif = "0.9.0"
 lazy_static = "1.1.0"
 openssl = { version = '0.10.11', optional = true }
 parking_lot = "0.6"
+rustc_version = "0.2.3"
```

Comment:
FWIW using this crate can be problematic sometimes until https://github.com/Kimundi/rustc-version-rs/issues/11 has been fixed, and in general determining just the minor version of rustc is [pretty lightweight](https://github.com/alexcrichton/proc-macro2/blob/79e40a113b51836f33214c6d00228934b41bd4ad/build.rs#L44-L61)